src/services/users/index.js

```diff
@@ -213,12 +213,47 @@ const getInvitedUserDetail = async (token) => {
 	
 }
 
+/**
+ * Generates a new password reset link
+ * @param {string} email - email of the user
+ * @returns {string} - password reset link
+ */
+const generateResetLink = async (email) => {
+	await getUserByEmail(email)
+
+	
+	const resetToken = await generateAccessToken(
+		{ data: { email }, 
+			expiry: '1h' })
+
+	const url =
+	 `${config("FRONTEND_BASE_URL")}/reset-password?token=${resetToken}`
+	return url
+}
+
+/**
+ * Sends a password reset link to the user
+ * @param {object} { email, url } - email of the user and password reset link
+ * @returns {Promise}
+ */
+const sendResetLink = async ({ email, url }) => {
```

Comment:
Looks cool but the file keeps growing, lets start separating the logic, have a file like `send-password-reset-link.js` and export it

after that, import it here and export it as part of the other functions